tooling/contracts/l1-contracts.mdx

```diff
@@ -0,0 +1,83 @@
+| OptimismMintableERC20FactoryProxy | [`0x6f0E4f1EB98A52EfaCF7BE11d48B9d9d6510A906`](https://eth.blockscout.com/address/0x6f0E4f1EB98A52EfaCF7BE11d48B9d9d6510A906) |
+| OptimismMintableERC20FactoryImpl | [`0x5493f4677a186f64805fe7317d6993ba4863988f`](https://eth.blockscout.com/address/0x5493f4677a186f64805fe7317d6993ba4863988f) |
+| OptimismPortalProxy | [`0xc5c5D157928BDBD2ACf6d0777626b6C75a9EAEDC`](https://eth.blockscout.com/address/0xc5c5D157928BDBD2ACf6d0777626b6C75a9EAEDC) |
+| OptimismPortalImpl | [`0x215a5ff85308a72a772f09b520da71d3520e9ac7`](https://eth.blockscout.com/address/0x215a5ff85308a72a772f09b520da71d3520e9ac7) |
+| PermissionedDelayedWETHProxy | [`0x9c314E8057025F2982aa4B3923Abd741A8e8DE91`](https://eth.blockscout.com/address/0x9c314E8057025F2982aa4B3923Abd741A8e8DE91) |
+| PermissionedDelayedWETHImpl | [`0x1e121e21e1a11ae47c0efe8a7e13ae3eb4923796`](https://eth.blockscout.com/address/0x1e121e21e1a11ae47c0efe8a7e13ae3eb4923796) |
+| PermissionedDisputeGame | [`0x25c2e07A24a74F9FA54f7CA5ddAfedB2264a5d02`](https://eth.blockscout.com/address/0x25c2e07A24a74F9FA54f7CA5ddAfedB2264a5d02) |
+| PreimageOracle | [`0x1fb8cdfc6831fc866ed9c51af8817da5c287add3`](https://eth.blockscout.com/address/0x1fb8cdfc6831fc866ed9c51af8817da5c287add3) |
+| ProtocolVersionsProxy | [`0x1b6dEB2197418075AB314ac4D52Ca1D104a8F663`](https://eth.blockscout.com/address/0x1b6dEB2197418075AB314ac4D52Ca1D104a8F663) |
+| ProtocolVersionsImpl | [`0x37e15e4d6dffa9e5e320ee1ec036922e563cb76c`](https://eth.blockscout.com/address/0x37e15e4d6dffa9e5e320ee1ec036922e563cb76c) |
+| ProxyAdmin | [`0x783A434532Ee94667979213af1711505E8bFE374`](https://eth.blockscout.com/address/0x783A434532Ee94667979213af1711505E8bFE374) |
+| SuperchainConfigProxy | [`0x95703e0982140D16f8ebA6d158FccEde42f04a4C`](https://eth.blockscout.com/address/0x95703e0982140D16f8ebA6d158FccEde42f04a4C) |
+| SuperchainConfigImpl | [`0xce28685eb204186b557133766eca00334eb441e4`](https://eth.blockscout.com/address/0xce28685eb204186b557133766eca00334eb441e4) |
+| SystemConfigProxy | [`0x89E31965D844a309231B1f17759Ccaf1b7c09861`](https://eth.blockscout.com/address/0x89E31965D844a309231B1f17759Ccaf1b7c09861) |
+| SystemConfigImpl | [`0x9c61c5a8ff9408b83ac92571278550097a9d2bb5`](https://eth.blockscout.com/address/0x9c61c5a8ff9408b83ac92571278550097a9d2bb5) |
+| SystemOwnerSafe | [`0x4092A77bAF58fef0309452cEaCb09221e556E112`](https://eth.blockscout.com/address/0x4092A77bAF58fef0309452cEaCb09221e556E112) |
+
+## Celo Sepolia Testnet
+
+| Contract | Address |
+| -------- | ------- |
+| AddressManager | [`0x8f0c6fc85a53551d87899ac2a5af2b48c793eb63`](https://sepolia.etherscan.io/address/0x8f0c6fc85a53551d87899ac2a5af2b48c793eb63) |
+| AnchorStateRegistryProxy | [`0xd73ba8168a61f3e917f0930d5c0401aa47e269d6`](https://sepolia.etherscan.io/address/0xd73ba8168a61f3e917f0930d5c0401aa47e269d6) |
```

Comment:
It's strange that the mainnet contract links use blockscout but the sepolia links use etherscan